server/data/data-api/src/main/java/org/apache/james/sieverepository/api/SieveRepositoryManagementMBean.java

```diff
@@ -36,4 +43,17 @@
 
     void removeQuota(String user) throws SieveRepositoryException;
 
+    void addActiveSieveScript(String user, String toFileName, String fromScriptFileName) throws Exception;
+
+    InputStream getActive(String user) throws ScriptNotFoundException, StorageException;
+
+    default String sanitizeScriptName(String fileName) {
```

Comment:
Can't yoy reuse this method in probes?

(i would consider you can make it static if needed :-) )

Reply:
Can reuse here but did not want one interface to depend on another unrelated interface just for one function.